frontend/src/LineGraph.res

```diff
@@ -95,7 +94,6 @@ let defaultOptions = (
         "axisLineWidth": 1.5,
         "axisLabelFormatter": Js.Null.fromOption(xLabelFormatter),
         "ticker": ticker,
-        "axisLabelWidth": 70,
```

Comment:
The last label is not overlapped anymore but still not perfectly aligned:

<img width="483" alt="Screenshot 2021-03-15 at 08 17 04" src="https://user-images.githubusercontent.com/5595092/111119854-ed591d00-856a-11eb-99f1-3e9f04c4f70c.png">

If we still want to improve it, I think we can either increase the width of the graph or use ` "axisLabelWidth": 50` which, from my test, looked good.

Reply:
This looks much better, thanks.

The default value seems to be `60` for `axisLabelWidth`. Which one looks better to you – 50 or 60?

Reply:
Just tested 50 – looks perfect. Should we use that?

Reply:
Indeed, 50 looks perfect. I was just wondering how it would look like for other graph dimensions. Omitting the property lets the library itself determine the label width - maybe it will better adapt to different graph dimensions?

Anyway, I will set it to 50 for now so that it looks better. We can still revisit that later on.